docs/synapse/userguides/analytical_model_tags_analysis.rst

```diff
@@ -5,164 +5,286 @@
+If Hypothesis 1 is true, we would not tag the FQDN. If Hypothesis 2 is true, we can simply assert that the FQDN is
+malicious (with a tag such as ``cno.mal``).
 
-Let’s say an analyst comes across the domain ``g00gle.com``, which bears a resemblance to the legitimate ``google.com`` domain. The mere similarity is not enough to determine whether the similar domain is malicious or used for malicious purposes. However, the analyst may have access to additional data (such as a phishing email with a link to a ``g00gle.com`` web site that prompts the user to enter their password). The analyst determines that the domain is malicious and likely intended for credential theft. Applying the tag ``cno.ttp.se.masq.google`` effectively means that the hypothesis "A threat actor created this domain to imitate Google for malicious purposes" has been assessed to be true.
+If Hypothesis 2 is true, deciding on Hypothesis 3 may be trickier. Does the FQDN "look like" anything
+familiar? It may "look like" Akamai CDN (content delivery network) but that's a bit of a stretch...maybe it is just
+a coincidence? Do we have any context around **how** the FQDN was used maliciously that might indicate that the
+threat actors wanted to mislead victims into thinking the FQDN was associated with Akamai?
+
+If we have enough evidence to support Hypothesis 3, we can apply a TTP tag such as ``cno.ttp.se.masq`` (``cno`` as
+our top-level tag, ``ttp`` for our TTP sub-tree, ``se`` for social engineering TTPs, and ``masq`` for masquerade).
 
 Individual Hypotheses to Broader Reasoning
 ++++++++++++++++++++++++++++++++++++++++++
 
-More complex hypotheses may not be explicitly annotated within the graph (that is, as tags applied to individual nodes), but may be supported (or refuted) by the presence of individual tags or combinations of tags on sets of nodes.
-
-For example, an analyst tracking Threat Cluster 12 believes (has a hypothesis) that they frequently register domains that imitate technology companies. In the absence of a detailed data modeling and tracking system (such as a Synapse hypergraph), such an assessment is often made based on an analyst’s "impression" of historical Threat Cluster 12 data / domains.
-
-A better way to make this determination based on tracked data and assessments would be to:
-
-- review all of the domains associated with Threat Cluster 12 (i.e., tagged ``cno.threat.t12``)
-- determine how many of those domains have ``cno.ttp.se.masq.*`` tags, and
-- determine the types of organizations represented by the ``masq`` tags (technology, media, government, etc.)
+You may not annotate **complex** hypotheses explicitly within Synapse (that is, as tags applied to individual nodes).
+But these larger hypotheses may be supported (or refuted) by individual tags or combinations of tags (the results
+of smaller, more focused hypotheses) on nodes.
+
```

Comment:
Is there a missing word / statement that prefaces this example?

Reply:
Revised